components/reddit/reddit.app.mjs

```diff
@@ -247,8 +358,16 @@ module.exports = {
       const results = [];
       let after = null;
       do {
-        const redditCommunities = await this.searchSubreddits(after, query);
-        const isNewDataAvailable = get(redditCommunities, [
+        const redditCommunities = await this.searchSubreddits({
+          after,
+          q: query,
+          limit: 100,
+          show_users: false,
+          sort: "relevance",
+          sr_detail: false,
+          typeahead_active: false,
+        });
+        const isNewDataAvailable = lodash.get(redditCommunities, [
```

Comment:
Could we reuse the imported `get` from `"lodash/get.js"` instead of importing `lodash` in this file?
```js
        const isNewDataAvailable = get(redditCommunities, [
```

Reply:
Oops, I forgot to remove this one :)